docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -65,6 +66,55 @@ Applications that target versions of .NET Framework starting with 4.7.1 can disa
+
+- Added support for the UIA Expand/Collapse pattern to the <xref:System.Windows.Forms.DateTimePicker> control
+- Added UIA support to the <xref:System.Wndows.Forms.MonthCalendar> control. Now assistive technology tools such as Narrator can navigate through the individual dates in the control.
+- Implemented Text Pattern support to all text based controls including <xref:System.Windows.Forms.TextBox>, <xref:System.Windows.Forms.MaskedTextBox>, <xref:System.Windows.Forms.PropertyGrid> edit control, <xref:System.Windows.Forms.DataGridViewTextBoxCell>, <xref:System.Windows.Forms.ToolStripTextBox>, <xref:System.Windows.Forms.DomainUpDown> controls.
+- <xref:System.Windows.Forms.ToolTips> now follow <a href='https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html'> WCAG2.1 guidelines</a> to be persistent, dismissable, and hoverable on Windows 11.
+
+**Various bug fixes for existing accessibility features**
+- Narrator can now focus on an empty <xref:System.Windows.Forms.DataGridView> control.
+- Updated the luminosity ratio to 3.5:1 for <xref:System.Windows.Forms.ToolStripButton> controls with a `ToolStripRenderMode` set to `System`
+- Fixed a case which could cause the screen reader JAWS to crash when reading the <xref:System.Windows.Forms.PropertyGrid> control.
+- Addressed an issue which caused screen readers to count hidden columns when announcing the column count in a <xref:System.Windows.Forms.DataGridView> control.
+- Addressed an issue cuaing the <xref:System.Windows.Forms.DataGridView> to ignore the font settings set in the `DataGridviewCellStyle` if the underlying form has a `Font` property that differs from the `DefaultFont`
+- Updated the `AccessibleName` property of the <xref:System.Windows.Forms.DataGridView> control's internal scroll bars to remove the text "ScrollBar".
+- Fixed the color of a `DataGridViewLinkCell` when the cell is selected.
+- Improved keyboard navigation in a <xref:System.Windows.Forms.ToolStrip> when the element is a `ToolStripComboBox` type.
+- Fixed an issue with the <xref:System.Windows.Forms.ComboBox> which could cause application crash under Accessibility Insights for Windows.
+- Updated the border color for the <xref:System.Windows.Forms.Button> to have more contrast in the default colors.
+- Ensured that there was a bounding rectangle reported to assitive technology for the <xref:System.Windows.Forms.ToolStripSeparator>
+- Enabled assistive technology tools to access the `ControlBox` buttons of a maximized MDI Child form.
+- Updated the background color of the <xref:System.Windows.Forms.ToolStripButton> in High Contrast mode.
+- <xref:System.Windows.Forms.DomainUpDown> control `AccessibleName` property has a new default value of an empty string. This will encourage developers to create a meanignful name rather than accepting a default of "Spinner"
```

Comment:
```suggestion
- The `AccessibleName` property of a <xref:System.Windows.Forms.DomainUpDown> control has a new default value of an empty string. The empty string will encourage developers to create a meaningful name rather than accepting a default of "Spinner".
```

Reply:
Also this is confusing. It says the default is an empty string, and then that it is "Spinner".

Reply:
Tried to rephrase, is this better?